OpenRA.Mods.Common/Traits/World/ActorMap.cs

```diff
@@ -510,6 +510,11 @@ public int AddCellTrigger(CPos[] cells, Action<Actor> onEntry, Action<Actor> onE
 			return id;
 		}
 
+		public List<CPos> TriggerPositions()
+		{
+			return cellTriggerInfluence.Keys.ToList();
```

Comment:
Consider returning `IEnumerable<CPos>` and avoiding the `ToList`? Seems like that would be a nicer signature to expose.